README.md

```diff
@@ -64,10 +64,11 @@ If you prefer direct control to singletons, manage ownership of the
 Additionally, this example demonstrates how to get a `context.Context`
 associated with any `opentracing.Span` instance.
 
-```
+```go
     func xyz(goCtx context.Context, ...) {
         ...
-        sp, goCtx := opentracing.JoinTrace("span_name", goCtx).AddToGoContext(goCtx)
+        goCtx, sp := opentracing.ContextWithSpan(
```

Comment:
FWIW, I was just tinkering with whether `Span` really needed that `AddToGoContext` helper method... the change here is not related to the main event in this PR.